portality/settings.py

```diff
@@ -124,8 +124,8 @@
 
 # List the features we want to be active (API v1 and v2 remain with redirects to v3)
 # ~~->API:Feature~~
-FEATURES = ['api1', 'api2', 'api3']
-VALID_FEATURES = ['api1', 'api2', 'api3']
+FEATURES = ['api1', 'api2', 'api3', 'api4', 'api_current']
+VALID_FEATURES = ['api1', 'api2', 'api3', 'api4', 'api_current']
```

Comment:
I wonder if we should remove api1 and api2 from the list of features now, it's been a while since we formally supported them?  @Steven-Eardley what do you think?